target/linux/mvebu/image/cortexa72.mk

```diff
@@ -1,3 +1,11 @@
+define Device/globalscale_mochabin
+  $(call Device/Default-arm64)
+  DEVICE_VENDOR := Globalscale
+  DEVICE_MODEL := MOCHAbin
+  DEVICE_DTS := armada-7040-mochabin
```

Comment:
You should be able to use "SOC := armada-7040" instead

Reply:
Will update it

Reply:
Done

Reply:
I do not see the change

Reply:
Sorry, pushed to the wrong branch.
Its pushed correctly now.